docs/gui.rst

```diff
@@ -522,3 +522,14 @@ Image I/O
     This function will create an instance of ``ti.GUI`` and show the input image on the screen.
 
     It has the same logic as ``ti.imwrite`` for different datatypes.
+
+.. function:: ti.imscale(img, w=512, h=None):
```

Comment:
```suggestion
.. function:: ti.imscale(img, w=512, h=512):
```
I would suggest to use a meaningful number in the parameter `h`, because it can guide our users better.

Reply:
How about `ti.imscale(img, w=512, h=w)`?

Reply:
Yes! I love that ;)